content/extraProblems.json

```diff
@@ -472,6 +472,31 @@
         "kind": "internal"
       }
     },
+    {
```

Comment:
idt we have to keep this one (cube world) since it doesn't have an internal solution